features/recognize.js

```diff
@@ -155,7 +174,7 @@ async function handleSlackError(client, message, error) {
 }
 
 async function handleGratitudeError(client, message, error) {
-  winston.info("Rejected gratitude request as invalid", {
+  winston.error("Rejected gratitude request as invalid", {
```

Comment:
I don't think this should be an error, as this is a user-side error (i.e. 400). In this code path, Gratibot is still functioning as expected. In my opinion this _could_ be a warning at most, but I think info is appropriate. Open to discussion of course.